cider/tests/utils/CMakeLists.txt

```diff
@@ -17,20 +17,12 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR})
-if(${CIDER_ENABLE_VELOX})
-  include_directories(
-    ${CMAKE_SOURCE_DIR}/thirdparty/velox/velox/substrait/proto)
-else()
-  include_directories(${CMAKE_SOURCE_DIR}/thirdparty/substrait/proto)
-endif()
-
-set(UTILS_SOURCE
+set(CIDER_TESTS_UTILS_SOURCE
     Utils.cpp DuckDbQueryRunner.cpp CiderQueryRunner.cpp CiderTestBase.cpp
     CiderBatchChecker.cpp CiderBenchmarkRunner.cpp)
 
-add_library(test_utils ${UTILS_SOURCE})
-target_link_libraries(test_utils util)
```

Comment:
just curious, where is `util` from? some mistakes from refactor?

Reply:
It originally from the logger library, then move to the util library.